chart/values.yaml

```diff
@@ -207,8 +204,8 @@ google:
   ## @param google.existingSecret.key Key name of the Google Credential file inside the secret
   ##
   existingSecret:
-    name: ""
-    key: ""
+    name: "google-serviceaccount"
```

Comment:
perhaps uploaded by mistake?

Reply:
I believe this is related with https://github.com/CartoDB/carto3-helm/pull/20/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R67 . In any case, i'd use a variable name more tied to `selfhosted` naming, may be `selfhosted-google-serviceaccount`.